src/openrct2/entity/Peep.cpp

```diff
@@ -250,6 +250,45 @@ void PeepUpdateAllBoundingBoxes()
     }
 }
 
+void Peep::UpdateWaitingAtCrossing()
+{
+    if (!IsActionInterruptable())
+    {
+        UpdateAction();
+        Invalidate();
+        if (!IsActionWalking())
+            return;
+    }
+
+    Action = PeepActionType::idle;
+    NextAnimationType = PeepAnimationType::watchRide;
+    SwitchNextAnimationType();
+
+    if (auto* guest = As<Guest>(); guest != nullptr)
```

Comment:
Split this over two lines:

```suggestion
    auto* guest = As<Guest>();
    if ( guest != nullptr)
```

Reply:
Resolved